classes/py1.html

```diff
@@ -6,9 +6,9 @@
 
 <div class="py1-wrapper">
     <div class="py1-sidebar">
-        <h1>how to get started in<br />
-        <span class="py1-link">python</span>
-        </h1>
+        <span class = "py1-smaller-margin-p">how to get started in</span>
```

Comment:
Is there a reason you removed this from being an `<h1>`? It doesn't seem like there's one on the page anymore!

Reply:
Was attempting to resolve a margins issue, but fixed it through using the `.no-margin` class and added the `<h1>` back!